frontend/app/.server/routes/helpers/protected-renew-route-helpers.ts

```diff
@@ -78,6 +79,13 @@ export interface ProtectedRenewState {
     mailingPostalCode?: string;
     mailingProvince?: string;
   };
+  readonly submissionInfo?: {
+    /**
+     * The UTC date and time when the application was submitted.
+     * Format: ISO 8601 string (e.g., "YYYY-MM-DDTHH:mm:ss.sssZ")
+     */
+    submittedOn: string;
+  };
```

Comment:
I don't think we need to keep this field in state as we're not displaying it anywhere or using it to call a service.